tests/unit/FunctionInjectorTest.php

```diff
@@ -28,7 +28,7 @@ public function testTemplate()
     public function testSave()
     {
         $this->funcInjector->save();
-        exec('php -l '.$this->getFileName(), $output, $code);
+        exec('php -l '.__FILE__, $output, $code);
```

Comment:
Hi, that was wrong. I fix it below commit. Initially, it should be `$this->funcInjector->getFileName()`